src/f32/mat4.rs

```diff
@@ -389,17 +389,17 @@ impl Mat4 {
     // TODO: make public at some point
     fn look_to_lh(eye: Vec3, dir: Vec3, up: Vec3) -> Self {
         let f = dir.normalize();
+        let s = up.cross(f).normalize();
+        let u = f.cross(s).normalize();
```

Comment:
You can get rid of that normalize. f and s are already normalized.

Reply:
Yup good point, will fix